.circleci/config.yml

```diff
@@ -43,7 +43,12 @@ jobs:
           command: npm run test:eslint
       - run:
           name: Unit Test Coverage
-          command: npm run test:coverage
+          command: npm run test:unit:coverage
+      - store_test_results:
+          path: test_results
+      - run:
```

Comment:
This should be its own job in the workflow, to run after and only, if linting + unit tests pass

Reply:
it will not get here if the unit test fails.

Reply:
It's easier to see exactly what failed if it's own step. If linting, unit tests and integration tests are separate, we can see right away from the PR which step failed, instead of having to go to CircleCi to find out what in "test" failed:
![Screen Shot 2019-11-19 at 3 41 37 PM](https://user-images.githubusercontent.com/6610308/69196340-1a1a3300-0ae3-11ea-9316-9db630f3c3a8.png)

Reply:
If we’re following the widget and sdk pipeline style, the CI testing is one “workflow” with multiple jobs executed in order of: static analysis, unit tests, integration tests. I think it is worthwhile to keep the same flow in the component repos. Having them as separate jobs also makes it easier on circle ci to just re-run the failed job.

Reply:
where do you do the linting?